code/modules/client/preference_setup/antagonism/01_candidacy.dm

```diff
@@ -2,6 +2,7 @@
 	var/list/never_be_special_role
 	var/list/be_special_role
 	var/auto_necroqueue = TRUE
+	var/auto_ert = FALSE
```

Comment:
this doesn't need to be an option, just selecting candidacy for the ERT roles is enough